client/signup/config/steps-pure.js

```diff
@@ -510,6 +540,14 @@ export function generateSteps( {
 			dependencies: [ 'themeSlugWithRepo' ],
 			delayApiRequestUntilComplete: true,
 		},
+
+		launch: {
+			stepName: 'launch',
+			apiRequestFunction: launchSiteApi,
+			dependencies: [ 'siteSlug' ],
+			providesDependencies: [ 'siteSlug' ],
+			delayApiRequestUntilComplete: true,
```

Comment:
I don't really understand the need for `delayApiRequestUntilComplete`. What happens if you remove it from here?

Reply:
Agree, this can be removed. I've also removed it from the `domains-launch` step. Implemented in [b2fcae0](https://github.com/Automattic/wp-calypso/pull/30242/commits/b2fcae034db8926a6854201af72a1966f6b32963).